modules/osdk-building-ansible-operator.adoc

```diff
@@ -0,0 +1,377 @@
+// Module included in the following assemblies:
+//
+// * operators/osdk-ansible.adoc
+
+[id='osdk-building-ansible-operator-{context}']
+= Building an Ansible-based Operator using the Operator SDK
+
+This procedure walks through an example of building a simple Memcached Operator
+powered by Ansible playbooks and modules using tools and libraries provided by
+the Operator SDK.
+
+.Prerequisites
+
+- Operator SDK CLI installed on the development workstation
+- Access to a Kubernetes-based cluster v1.11.3+ (for example {product-title} 4.0)
+using an account with `cluster-admin` permissions
+- link:https://kubernetes.io/docs/tasks/tools/install-kubectl/[`kubectl`] v1.11.3+
+(can alternatively use `oc`)
+- link:https://docs.ansible.com/ansible/latest/index.html[`ansible`] v2.6.0+
+- link:https://ansible-runner.readthedocs.io/en/latest/install.html[`ansible-runner`] v1.1.0+
+- link:https://github.com/ansible/ansible-runner-http[`ansible-runner-http`] v1.0.0+
+
+.Procedure
+
+. *Create a new Operator project,* either namespace-scoped or cluster-scoped,
+using the `operator-sdk new` command. Choose one of the following:
+
+.. A _namespace-scoped Operator_ (the default) watches and manages resources in a
+single namespace. Namespace-scoped operators are preferred because of their
+flexibility. They enable decoupled upgrades, namespace isolation for failures
+and monitoring, and differing API definitions.
++
+To create a new Ansible-based, namespace-scoped
+`memcached-operator` project, use the following command command:
++
+----
+$ operator-sdk new memcached-operator \
+    --api-version=cache.example.com/v1alpha1 \
+    --kind=Memcached \
+    --type=ansible
+$ cd memcached-operator
```

Comment:
Just wanting to double check that "cd memcached-operator" is supposed to be included here.

Reply:
Yes, but it wasn't in the cluster-scoped version of the command, so I added it there and added "and change to its directory" to the steps' text for accuracy.